boot.mount

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Before=local-fs.target
+RequiresOverridable=systemd-fsck@dev-disk-by\x2dlabel-ostree\x2dboot.service
+After=systemd-fsck@dev-disk-by\x2dlabel-ostree\x2dboot.service
+
+[Mount]
+What=/dev/disk/by-label/ostree-boot
+Where=/boot
+Type=ext2
+
+[Install]
+WantedBy=dev-disk-by\x2dlabel-ostree\x2dboot.device
```

Comment:
I don't actually think this is required. I believe that systemd will take the `What`, convert it to a device unit, and add it `BindsTo` and `WantedBy`. Can you try with and without? You should just be able to do `systemctl daemon-reload && systemctl show boot.mount` to see differences.